addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1379,12 +1379,12 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
      * @override
      */
     close: function () {
-        this._super(...arguments);
         if (this._customColorValue && this._customColorValue !== this._value) {
             this._value = this._customColorValue;
             this._customColorValue = false;
             this._onUserValueChange();
         }
+        this._super(...arguments);
```

Comment:
I kinda have the same question as commit n°2 about this, I don't remember why we have that code :scream: